src/openlrr/game/object/ObjectRecall.cpp

```diff
@@ -22,27 +25,163 @@ LegoRR::ObjectRecall_Globs & LegoRR::objectRecallGlobs = *(LegoRR::ObjectRecall_
 #pragma region Functions
 
 // <LegoRR.exe @00459450>
-//void __cdecl LegoRR::ObjectRecall_StoreMiniFigure(const LegoObject* liveObj);
+void __cdecl LegoRR::ObjectRecall_StoreMiniFigure(const LegoObject* liveObj)
+{
+
+	//Check type is correct, ability flags are set, and name is custom
+	if ((liveObj->type == LegoObject_MiniFigure) &&
+		(((liveObj->abilityFlags != ABILITY_FLAG_NONE || (liveObj->objLevel != 0)) || (liveObj->customName != nullptr)))) {
+		if (objectRecallGlobs.recallUsed == objectRecallGlobs.recallCapacity) {
+			/* More space is needed, allocate 10 extra slots */
+			//FC, should this be OBJECT_MAXRECORDOBJECTS?
```

Comment:
This is a list capacity increase. Capacity is used to avoid reallocating the list every time an item is added be preparing for up to X more items being added. In this case, whenever the list is full, it makes room for 10 more items (9 after adding the item).

Also when signing a comment, it's preferred to use a dash followed by your name at the end of the comment (or wrapped on a new line if the last line is long enough/or you think a newline would be easier to read). Whether you want to use initials, first name, last name, full name, or username is up to you. But if you have no preference, then the easier name to identify is preferred (only if you're comfortable with that of course).
```cpp
// comment that may span
// over multiple lines. -FC
```